subprojects/common/graphite/src/main/kotlin/com/avito/android/graphite/GraphiteSender.kt

```diff
@@ -0,0 +1,49 @@
+package com.avito.android.graphite
+
+import java.io.DataOutputStream
+import java.net.Socket
+
+
+interface GraphiteSender {
+
+    /**
+     * Incubating
+     */
+    fun send(metric: GraphiteMetric)
+
+    class Impl(
+        private val config: GraphiteConfig,
+        private val logger: (String, Throwable?) -> Unit
```

Comment:
It's look like tech debt. Let's use normal logger

Reply:
fixed